packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -72,7 +82,7 @@ export class DataProxyEngine extends Engine {
 
     const [host, apiKey] = this.extractHostAndApiKey()
     this.remoteClientVersion = P.then(() => getClientVersion(this.config))
-    this.headers = { Authorization: `Bearer ${apiKey}` }
+    this.headers = { Authorization: `Bearer ${apiKey}`, 'X-capture-traces': 'true' }
```

Comment:
This should not be sent unconditionally, but only when query logs are enabled in the client